app/src/main/java/com/x8bit/bitwarden/data/platform/util/ServerVersionUtils.kt

```diff
@@ -0,0 +1,51 @@
+package com.x8bit.bitwarden.data.platform.util
+
+import com.x8bit.bitwarden.data.platform.datasource.disk.model.ServerConfig
+import kotlin.text.split
+import kotlin.text.toIntOrNull
+
+private const val VERSION_SEPARATOR = "."
+private const val SUFFIX_SEPARATOR = "-"
+
+/**
+ * Checks if the server version is greater than another provided version, returns true if it is.
+ */
+fun isServerVersionAtLeast(serverConfig: ServerConfig?, version: String?): Boolean {
```

Comment:
Last question, why would the version ever be null?

Reply:
It's shouldn't be, but I felt weird skipping the null check further down just to satisfy the IDE so I made it nullable. Thoughts?

Reply:
I think we can make `version` nonnull and just do the empty check